google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableInstanceAdminClientTests.java

```diff
@@ -1034,6 +1034,135 @@ public void testCreateAppProfileAddPriority() {
+            .thenReturn(ApiFutures.immediateFuture(expectedResponse));
+
+    // Execute
+    AppProfile actualResult =
+            adminClient.createAppProfile(
+                    CreateAppProfileRequest.of(INSTANCE_ID, APP_PROFILE_ID)
+                            .setDescription("my description")
+                            .setRoutingPolicy(MultiClusterRoutingPolicy.withRowAffinity("cluster-id-1", "cluster-id-2")));
+
+    // Verify
+    assertThat(actualResult).isEqualTo(AppProfile.fromProto(expectedResponse));
+  }
+
+  @Test
+  public void testCreateAppProfileAddRowAffinityAddMultipleClusterIdsWithList() {
+    // Setup
+    Mockito.when(mockStub.createAppProfileCallable()).thenReturn(mockCreateAppProfileCallable);
+
+    com.google.bigtable.admin.v2.CreateAppProfileRequest expectedRequest =
+            com.google.bigtable.admin.v2.CreateAppProfileRequest.newBuilder()
+                    .setParent(NameUtil.formatInstanceName(PROJECT_ID, INSTANCE_ID))
+                    .setAppProfileId(APP_PROFILE_ID)
+                    .setAppProfile(
+                            com.google.bigtable.admin.v2.AppProfile.newBuilder()
+                                    .setDescription("my description")
+                                    .setMultiClusterRoutingUseAny(
+                                            com.google.bigtable.admin.v2.AppProfile.MultiClusterRoutingUseAny
+                                                    .newBuilder()
+                                                    .addAllClusterIds(ImmutableList.of("cluster-id-1", "cluster-id-2"))
+                                                    .setRowAffinity(com.google.bigtable.admin.v2.AppProfile.MultiClusterRoutingUseAny.RowAffinity.getDefaultInstance())))
+                    .build();
+
+    com.google.bigtable.admin.v2.AppProfile expectedResponse =
+            com.google.bigtable.admin.v2.AppProfile.newBuilder()
+                    .setName(APP_PROFILE_NAME)
+                    .setDescription("my description")
+                    .setMultiClusterRoutingUseAny(
+                            com.google.bigtable.admin.v2.AppProfile.MultiClusterRoutingUseAny.newBuilder()
+                                    .addAllClusterIds(ImmutableList.of("cluster-id-1", "cluster-id-2"))
+                                    .setRowAffinity(com.google.bigtable.admin.v2.AppProfile.MultiClusterRoutingUseAny.RowAffinity.getDefaultInstance()))
+                    .build();
+
+    Mockito.when(mockCreateAppProfileCallable.futureCall(expectedRequest))
+            .thenReturn(ApiFutures.immediateFuture(expectedResponse));
+
+    // Execute
+    AppProfile actualResult =
+            adminClient.createAppProfile(
+                    CreateAppProfileRequest.of(INSTANCE_ID, APP_PROFILE_ID)
+                            .setDescription("my description")
+                            .setRoutingPolicy(MultiClusterRoutingPolicy.withRowAffinity("cluster-id-1", "cluster-id-2")));
```

Comment:
Is this the same as the test above?

Reply:
It is similar. The one above adds the cluster_ids one at a time via "addClusterIds". This test adds all the cluster_ids in one list via "addAllClusterIds"

Reply:
I think that's testing the generated proto instead of testing our wrapper class. We can probably just keep one of the 2 test.

Reply:
Okie - removed this one.